### DISCUSSION TITLE
Doubt: Is there a possibility to export or save data to restore later?

### DISCUSSION THREAD

**Discussion by guiquimthe** [ezra-bible-app/ezra-bible-app#585] [Q&A] [answered]
Hello my brothers! Good morning,

If there is a problem with my computer, can I export and import my tags and notes from Ezra Bible App? Is there a way to save my data? Or is this not implemented yet?

Thank you very much,

In Christ,

**Comment by guiquimthe** [comment]
Please, if that is enough, please disregard the question.

https://ezrabibleapp.net/howto/synchronize_database_with_multiple_devices

**Comment by tobias-klein** [comment, marked as answer]
This is a good question! @guiquimthe
If you simply need a backup, then it's enough to save the database for somewhere and I think you find enough instructions in the page you linked above.

If you mean an export of data in a more accessible form like CSV - that's still on the todo list of things to implement, see #508.

**Reply by guiquimthe** [reply]
Thank you very much for the feedback!
